v2/internal/controllers/crd_web_test.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	resources "github.com/Azure/azure-service-operator/v2/api/resources/v1beta20200601"
+	"github.com/Azure/azure-service-operator/v2/api/web/v1beta20220301"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/onsi/gomega"
+)
+
+func Test_Web_ServerFarm_CRUD(t *testing.T) {
+	t.Parallel()
+
+	tc := globalTestContext.ForTest(t)
+
+	rg := tc.CreateTestResourceGroupAndWait()
+
+	serverfarm := newServerFarm(tc, rg)
+
+	tc.CreateResourceAndWait(serverfarm)
```

Comment:
Are any of the properties of server farm updatable after creation? It might be a good idea to test that updates work as well as creation.

Reply:
Done!